### ISSUE TITLE
LB Nodes - cropped labels

### ISSUE THREAD

**Issue by ppaawweeuu** [ladybug-tools/ladybug-blender#14]
All LB Nodes looks similar, and i understand that it is begining but labels are often hidden and unreadable without resizing.
It is not critical, not bug, but very annoying and discouraging.
Can we change proportions of that generic node? Or some other generic/temporary solution to stop hidding that labels.
3rd example is photoshoped.

![nodes_width](https://user-images.githubusercontent.com/61344631/95014503-b4c22880-0647-11eb-985b-109c01f55b38.JPG)

**Comment by vicdoval** [comment]
Another idea to improve readability would be 
```
a= '_prop_name_'
b= a.title().replace('_',' ')[1:-1]
print(b)
>>'Prop Name'
```

**Comment by Moult** [comment]
@vicdoval apparently the use of underscores is a ladybug convention. As these nodes are generated from upstream, these conventions are maintained. An underscore prefix means that it is a required, non-optional input. An underscore suffix means that if no input is provided, a default is hardcoded. This behaviour is consistent across Ladybug on other platforms, so needs to be maintained in Blender (of course, we may choose to visualise it in a nicer way than underscores! Maybe some inspiration taken from https://github.com/nortikin/sverchok/pull/3312)

@ppaawweeuu completely agree it is not readable right now. I'll investigate to see if there is a way to improve it.

**Comment by mostaphaRoudsari** [comment]
As @Moult explained the _ helps the users to identify required inputs versus the optional inputs quickly without the need to read the documentations. At least until the Blender plugin is stable we should keep them as is. Changing them might introduce unexpected bugs.

**Comment by seamoo** [comment]
I would add that a feature I really liked in learning LB in GH was the text helpers when you hover over input/output text in a component. Especially now LB has gone to 1.0.0 I find new inputs and outputs or things have been moved our relabelled. I'm finding I need to have GH open to understand what these mean and what I may need to connect. .  

Some users may / will be coming to LB for the first time and may not have access to Rhino license to run GH/LB. In this case they will be at a disadvantage. Implementing a similar feature in Blender or by creating stand along wiki to describe the node and its input and output functions would be invaluable for new users. 

I did find the very helpful 'examples' incuded in Sverchok. Could examples be added for LB here to allow users understand typial builds for common problems. This would be helpful for new users or thoes coming from GH to understand Sverchok 'A Number', 'True' and 'Notes' and how they relate to inputs in Blender GH. 

![Examples](https://user-images.githubusercontent.com/25322402/95227427-bcd2c180-081b-11eb-9b54-ff3a9e1c5fc1.JPG)
